SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com|bolgehastanesi.com
 !
+||common.doosanmagazine.com/reciveLogs.php
```

Comment:
What about:
`||vogue.co.kr/wp-content/themes/Avada-Child-Theme/js/GA.min.js`
instead

Reply:
It looks not bad.
However, the rule can be useful on other website (e.g. https://www.allurekorea.com/ ).

Reply:
@piquark6046 
with this rule `/Avada-Child-Theme/js/GA.*js$domain=allurekorea.com|vogue.co.kr`
_request common.doosanmagazine.com/reciveLogs.php_ doesn't exist so let's replace `||common.doosanmagazine.com/reciveLogs.php` with `/Avada-Child-Theme/js/GA.*js$domain=allurekorea.com|vogue.co.kr`.

Reply:
Pushed.